security_considerations.md

```diff
@@ -10,40 +10,55 @@ if an OEM or supplier deems it necessary. In this section, we discuss several of
+First, they SHOULD diligently remove obsolete images from new versions of Targets metadata files uploaded to the Image repository. This can prevent attackers who compromise the Director repository from being able to choose these obsolete images for installation. This method has a downside in that it complicates the update process for vehicles that require an intermediate update step. For example, an ECU has previously installed image A, and C is the latest image it should install. However, the ECU should install image B before it installs C, and B has already been removed from the Targets metadata on the Image repository in order to prevent / limit rollback attacks. Thus, the OEM and / or its suppliers SHOULD balance between the requirement to remove obsolete images from the Targets metadata, and the need to support ECUs that require installation of these obsolete images before updating to the latest images.
 
-Second, they SHOULD decrease the expiration timestamps in the targets metadata uploaded to the image repository. This is done so that these targets metadata expire more quickly, thus preventing attackers who compromise the director repository from being able to choose these obsolete images. This method has the downside that these targets metadata now need to be updated more quickly. This will not cause accidental freeze attacks as long as an ECU can update both the time from the time server, and metadata from the image repository. Accidental freeze attacks can happen if, for some reason, the ECU is able to update metadata, but not the time. In this unlikely event, the ECU is able to continue working with the previously installed image, but it would be unable to update to the latest image. The director repository can detect this unlikely event using the vehicle version manifest. In this case, the OEM MAY require the owner of the vehicle to diagnose the problem at the nearest dealership or authorized mechanic.
+Second, they SHOULD decrease the expiration timestamps in the Targets metadata uploaded to the Image repository. This is done so that these Targets metadata expire more quickly, thus preventing attackers who compromise the Director repository from being able to choose these obsolete images. This method also has a downside, since these Targets metadata now need to be updated more quickly. To prevent accidental freeze attacks, an ECU needs to be able to update both the time from the time server, and metadata from the Image repository.  In this unlikely event that the ECU is able to update metadata, but not the time, it can continue working with the previously installed image, but would be unable to update to the latest image. The Director repository can detect this unlikely event using the vehicle version manifest. In this case, the OEM MAY require the owner of the vehicle to diagnose the problem at the nearest dealership or authorized mechanic.
```

Comment:
```suggestion
Second, they SHOULD decrease the expiration timestamps in the Targets metadata uploaded to the Image repository. This is done so that these Targets metadata expire more quickly, thus preventing attackers who compromise the Director repository from being able to choose these obsolete images. This method also has a downside, since these Targets metadata now need to be updated more quickly. To prevent accidental freeze attacks, an ECU needs to be able to update both the time from the Time Server, and metadata from the Image repository.  In this unlikely event that the ECU is able to update metadata, but not the time, it can continue working with the previously installed image, but would be unable to update to the latest image. The Director repository can detect this unlikely event using the vehicle version manifest. In this case, the OEM MAY require the owner of the vehicle to diagnose the problem at the nearest dealership or authorized mechanic.
```